### PR TITLE
Sharing docs

### PR DESCRIPTION
**Pull request by kjesvale**

### CODE REVIEW

File: src/docbkx/en/content/maintenance/common/mtn_share.xml [hunk 1]
```diff
@@ -22,28 +25,45 @@
...
+                    <para><emphasis role="bold">Can view only</emphasis>: The access group can view
+                        the object.</para>
                 </listitem>
                 <listitem>
-                    <para><emphasis role="bold">No access</emphasis>: The public won't have access
-                        to the object. This setting is only applicable to <emphasis role="bold"
-                            >Public access</emphasis>.  </para>
+                    <para><emphasis role="bold">No access</emphasis> (only applicable to <emphasis
+                            role="bold">Public access</emphasis>): The public won't have access to
+                        the object.</para>
```
Comment (stian-sandvold) [inline src/docbkx/en/content/maintenance/common/mtn_share.xml]: Maybe I am misunderstanding "Public access", but that's a property of the sharing settings - correct? So "Public access" is an access group? Maybe explain/clear up what Public access is before the ordered list?

File: src/docbkx/en/content/maintenance/common/mtn_share.xml [hunk 2]
```diff
@@ -22,28 +25,45 @@
...
+                </listitem>
+            </itemizedlist>
+        </listitem>
+        <listitem>
+            <para>Change data sharing settings for the users and user groups you want to modify.</para>
```
Comment (stian-sandvold) [inline src/docbkx/en/content/maintenance/common/mtn_share.xml]: Are users and user-groups also access groups? or is there just 1 access group, public access? If they are the same, why do we have 2 steps for each, if they are effectively the same? If they are not the same, why are the sub-items basically the same as for access groups?

File: src/docbkx/en/content/sharing/sharing.xml [hunk 3]
```diff
@@ -23,10 +23,16 @@ Future releases: Legacy sharing dialog will be replaced with new in the analytic
...
       by clicking on the &quot;Sharing settings&quot; button next to each report in the list. Implementers can use this feature to allow access to certain objects to only certain user groups. Users can use the feature to decide who they would like to share objects (such as pivot tables, charts, dashboards, etc) with. </para>
-    <para>If sharing is supported for a particular class of objects, a dialog will be available called &quot;Sharing settings&quot;, usually available by clicking on the name of the object or in the analytics tools, through an icon (Share with other people). Once you have accessed the sharing settings for the object you wish to share, a dialog similar to the one below will be shown. </para>
+    <para>If sharing is supported for a particular class of objects, a dialog will be available
+      called &quot;Sharing settings&quot;, usually available by clicking on the name of the object
+      or in the analytics tools, through an icon (Share with other people). Once you have accessed
+      the sharing settings for the object you wish to share, a dialog will be shown. There are
+      currently two different sharing dialogs in DHIS2; for instance, the
```
Comment (stian-sandvold) [inline src/docbkx/en/content/sharing/sharing.xml]: DHIS2. For instance,

File: src/docbkx/en/content/sharing/sharing.xml [hunk 4]
```diff
@@ -35,9 +41,12 @@ Future releases: Legacy sharing dialog will be replaced with new in the analytic
         </mediaobject>
       </screenshot>
     </para>
+    <para>Newer apps, such as the <emphasis>Maintenance</emphasis>, the new <emphasis>Dashboard
```
Comment (stian-sandvold) [inline src/docbkx/en/content/sharing/sharing.xml]: Use same names for all apps, IE. Maintenance *APP*, the new Dashboard app, and the new Maps app.

File: src/docbkx/en/content/sharing/sharing.xml [hunk 5]
```diff
@@ -35,9 +41,12 @@ Future releases: Legacy sharing dialog will be replaced with new in the analytic
...
+      </emphasis>app and the new <emphasis>Maps</emphasis> app, use a sharing dialog similar to the
+      image below:</para>
     <para>
       <screenshot>
-        <title>The sharing dialog in the Maintenance app</title>
+        <title>The new sharing Dialog</title>
```
Comment (stian-sandvold) [inline src/docbkx/en/content/sharing/sharing.xml]: "sharing dialog"

File: src/docbkx/en/content/sharing/sharing.xml [hunk 6]
```diff
@@ -50,10 +59,10 @@ Future releases: Legacy sharing dialog will be replaced with new in the analytic
     <para>Next to &quot;Public
       access&quot; you can choose your public access option: &quot;None&quot;, &quot;Can view&quot; or &quot;Can edit and view&quot;.
  Public access refers to users which are logged into the system.       Edit also implies deleting the report. </para>
-    <para>To share with a group, simply start typing the name of
-      the group and the &quot;Search for user groups&quot; input field and select your desired group. Click on
-      the &quot;+&quot; icon next to the input field to share with that group. For each group you can set an
-      access option, similar to public access.</para>
+    <para>To share with a group, simply start typing the name of the group and the &quot;Search for
```
Comment (stian-sandvold) [inline src/docbkx/en/content/sharing/sharing.xml]: name of the group in the "Search for user groups" input field

File: src/docbkx/en/content/sharing/sharing.xml [hunk 7]
```diff
@@ -50,10 +59,10 @@ Future releases: Legacy sharing dialog will be replaced with new in the analytic
...
+      user groups&quot; input field and select your desired group. Click on the user or user group
```
Comment (stian-sandvold) [inline src/docbkx/en/content/sharing/sharing.xml]: Will the "Search for user groups" input return both Users and UserGroups? Consider re-labeling the field to "Search for users and user groups" in that case

File: src/docbkx/en/content/maintenance/common/mtn_share.xml (hunk 2 above)
Comment (kjesvale) [reply src/docbkx/en/content/maintenance/common/mtn_share.xml]: I'm changing to "access groups" here as well. There should be one itemizedList for metadata sharing settings and one for data sharing settings. "Access group" simply refers to a "rule", i.e. the configuration given in the dropdown menu, and covers both public access, user access and user group access.

**Pull request merged**